docs/DEPLOYMENT-VERIFICATION.md

```diff
@@ -0,0 +1,168 @@
+# Test Pilot Test Plan
+
+## ENVIRONMENTS:
```

Comment:
I do, added to the wall of verification steps below, so people remember to confirm after installing the add-on.